### PR TITLE
ID_Lookup_Controller & Suppression API Examples Update

### PR DESCRIPTION
**Pull request by crugas**
Updated Python example and added SHELL example for find_by_id resources endpoint thanks to Dave Mayo - thanks Dave! Reformatted examples to match code standards.

Also added examples to suppression.rb.

## Description
Updated python example for find_by_id resources endpoint to not have to use urllib.parse import. Dave pointed me in the right direction on how to structure the request using archivessnake. Also added SHELL example for same endpoint. Reformatted the code to make it easier to read. Also added shell and python examples to suppression.rb - forgot to create a separate branch for that, sorry!

## Related JIRA Ticket or GitHub Issue
API Documentation subgroup

## How Has This Been Tested?
No testing. Just added/modified examples.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have authority to submit this code.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: backend/app/controllers/id_lookup_controller.rb [hunk 1]
```diff
@@ -2,35 +2,83 @@ class ArchivesSpaceService < Sinatra::Base
 
   Endpoint.get('/repositories/:repo_id/find_by_id/resources')
     .description("Find Resources by their identifiers")
+    .example("shell") do ||
```
Comment (donaldjosephsmith) [inline backend/app/controllers/id_lookup_controller.rb:5]: Is this empty `||` meant to be here? It doesn't seem to do any harm but looked odd to me.

Comment (crugas) [reply backend/app/controllers/id_lookup_controller.rb:5]: Looks like that slipped through. RubyMine has been adding those pipes by default. I can remove it and resubmit the pull request if you would like. I can double check if there's any other places those are showing up.

Comment (donaldjosephsmith) [reply backend/app/controllers/id_lookup_controller.rb:5]: Gotcha - IDEs are almost _too_ helpful these days, if you ask me 😆 Anyway, yes, that would be great if it isn't too much trouble.

Comment (crugas) [reply backend/app/controllers/id_lookup_controller.rb]: @donaldjosephsmith Removed those pipes, fixed some indenting issues, and others. Would it be possible to squash commits and resubmit this pull request by any chance?

**Review by donaldjosephsmith — APPROVED** [review]

**Pull request merged**